src/main/java/Main/ArgumentParser.java

```diff
@@ -1,26 +1,22 @@
 package Main;
 
-import java.util.LinkedList;
 import java.util.List;
 
 import Arguments.GrepperArguments;
-import Arguments.SpeechToTextArguments;
+import Arguments.PrintArguments;
+import Arguments.SearchArguments;
+import Arguments.TranscriptArguments;
+import Transcription.VoskAdapter;
 import picocli.CommandLine.Parameters;
 import picocli.CommandLine.Option;
 import picocli.CommandLine;
 
-import Search.Grepper;
-import SpeechToText.SpeechToText;
-import SpeechToText.VoskSpeechToText;
-
 public class ArgumentParser {
```

Comment:
Builder patterns in this file are inconsistent; Remove the variables from these builder pattern methods, and just pass the output of the methods directly. 

Also add a getSpeechToText method.

Reply:
Added getSpeechToText method, passed output directly.